test/DI.Tests/ServiceCollectionServiceExtensionsTest.cs

```diff
@@ -211,11 +211,14 @@ public static TheoryData TryAddImplementationTypeData
             ServiceLifetime expectedLifetime)
         {
             // Arrange
-            var collection = new ServiceCollection();
-            collection.Add(ServiceDescriptor.Transient(expectedServiceType, expectedServiceType));
+            var collection = new ServiceCollection
+            {
+                ServiceDescriptor.Transient(expectedServiceType, expectedServiceType)
+            };
 
             // Act
             addAction(collection);
+            collection.TryAdd(ServiceDescriptor.Describe(expectedServiceType, expectedImplementationType, expectedLifetime));
```

Comment:
xunit1026: "unused theory parameters". The MemberData has items in it that aren't really needed for the test. We could either duplicate part of the memberdata into a colleciont that doesn't have these extra parameters, or use them.